### ISSUE TITLE
Suggestion - Earthpol - Add occupier info to dynmap

### ISSUE THREAD

**Issue by Goosius1** [Nebula-MC/Nebula#2]
**What server is this for? (Network Wide, EarthPol, Skyblock, Oneblock etc)**
- Earthpol

**Is your feature request related to a problem? Please describe.**
- The dynmap does not show who is the occupier of towns

**Describe the solution you'd like**
- Add occupier info to the dynmap
- Here is the suggestion from the install guide:
```
 <br/><span style="font-weight:bold;">Occupier&colon; %occupier%</span>
```

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by 0xBit-MC** [comment]
In addition to the fix in #1. I'll add this as well when I move the Dynmap.

**Comment by 0xBit-MC** [comment]
Suggestion was added.
